doc/content/en/docs/++version++/Editor Support/_index.md

```diff
@@ -0,0 +1,175 @@
+---
+title: "Editor Support"
+linkTitle: "Editor Support"
+weight: 209
+---
+
+<!--
+
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+   https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+
+-->
+
+## Overview
+
+When editing an Avro schema, everyone has a preferred editor. Some use an IDE, like IntelliJ, Pycharm, RustRover or
+Visual Studio, while others prefer a more humble text editor like Atom/Pulsar, Emacs or Vim.
```

Comment:
```suggestion
Visual Studio, while others prefer a more humble text editor like Pulsar, Emacs or Vim.
```
IMO there is no need to mention Atom anymore since it is discontinued.

Reply:
Oops... I'm not used to the app... iI'll fix this tomorrow